modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -1518,8 +1700,10 @@ PERF_TEST_P_(RemapPerfTest, TestPerformance)
     }
 
     // Comparison ////////////////////////////////////////////////////////////
-    // FIXIT unrealiable check: EXPECT_EQ(0, cv::countNonZero(out_mat_ocv != out_mat_gapi));
-    EXPECT_EQ(out_mat_gapi.size(), sz_in);
+    {
+        EXPECT_TRUE(cmpF(out_mat_ocv, out_mat_gapi));
+        EXPECT_EQ(out_mat_gapi.size(), sz);
```

Comment:
Please leave only one `EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));` and remove the rest.

Reply:
Applied, thanks